perl/extensions.bzl

```diff
@@ -1,10 +1,17 @@
 """Entry point for extensions used by bzlmod."""
 
-load("@rules_perl//perl:deps.bzl", "perl_repos")
+load("@rules_perl//perl:deps.bzl", "perl_repos", "perl_rules_dev_dependencies")
```

Comment:
I don't think this can work: This file needs to be loadable by downstream consumers to run the extensions. But they won't be able to see the dev dependencies

Reply:
Good point, let me add a test to make sure it works